src/constants.json

```
@@ -0,0 +1,16 @@
+{
+    "random": "Check this box to enable random safety car events.",
+    "random_max_occ": "The maximum number of times a safety car event could potentially occur (not guaranteed); applies to only random safety cars.",
+    "random_prob": "The probability of a single safety car deployment, irrespective of the maximum random occurrences. A value of 0.1 indicates a 10% chance of a safety car being deployed.",
+    "stopped": "Check this box to enable a safety car event when cars are stopped on the track.",
```

```suggestion
    "stopped": "Check this box to enable safety car events when cars are stopped on the track.",
```
An adjustment so users know more than one of these events is possible in a race.